lib/Sema/DeclSpec.cpp

```diff
@@ -1051,6 +1053,13 @@ bool DeclSpec::setSpecItypeforany(SourceLocation Loc, const char *&PrevSpec, uns
   return false;
 }
 
+bool DeclSpec::setUnpackSpec(SourceLocation Loc, const char *&PrevSpec, unsigned &DiagID) {
+  // TODO: add error handling
```

Comment:
It would be good to add error handling now so that we don't have to revisit this file later.

Reply:
I can't think of any error conditions that apply. I think the comment was just wrong.